src/sgame/sg_main.cpp

```diff
@@ -260,6 +260,7 @@ Cvar::Cvar<bool> g_bot_attackStruct("g_bot_attackStruct", "whether bots target b
 Cvar::Cvar<float> g_bot_fov("g_bot_fov", "bots' \"field of view\"", Cvar::NONE, 125);
 Cvar::Cvar<int> g_bot_chasetime("g_bot_chasetime", "bots stop chasing after x ms out of sight", Cvar::NONE, 5000);
 Cvar::Cvar<int> g_bot_reactiontime("g_bot_reactiontime", "bots' reaction time to enemies (milliseconds)", Cvar::NONE, 500);
+Cvar::Cvar<int> g_bot_defaultFill("g_bot_defaultFill", "fills both teams with that number of bots at start of game", Cvar::NONE, 3);
```

Comment:
The commit message here seems wrong? Can we make the code and commit message consistent?

Reply:
Certainly, and I got caught myself at it for the draft I started to only add a way by GUI to do it.
Understand though that the cvar's name was changed after a request, so that it currently just break the average convention (when it comes to bots at least).
I'm not against rewording, but not until the PR is accepted.

Reply:
I'll rebase to get less commits when things will be ready, and will update the commit messages to be more meaningful in the go.